kuma/javascript/src/ga-provider.jsx

```diff
@@ -1,12 +1,8 @@
 // @flow
 import * as React from 'react';
+import { useContext, useEffect, useState } from 'react';
 
-type GAOptions = { [string]: any };
-type GAFunction = (
-    string,
-    string | GAOptions,
-    ...Array<string | number | GAOptions>
-) => void;
+type GAFunction = (...any) => void;
```

Comment:
It turned out to be too hard to accurately capture the type of the ga() function, and trying to do so was causing more flow problems than it was worth. So I took this easy way out.